apis/v1alpha1/generated.proto

```diff
@@ -1201,20 +1211,28 @@ message TCPRouteStatus {
 // - aws: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/create-https-listener.html#describe-ssl-policies
 // - azure: https://docs.microsoft.com/en-us/azure/app-service/configure-ssl-bindings#enforce-tls-1112
 message TLSConfig {
-  // CertificateRefs is a list of references to Kubernetes objects that each
-  // contain an identity certificate.  The host name in a TLS SNI client hello
-  // message is used for certificate matching and route host name selection.
-  // The SNI server_name must match a route host name for the Gateway to route
-  // the TLS request.  If an entry in this list omits or specifies the empty
+  // Mode defines the TLS behavior for the TLS session initiated by the client.
+  // There are two possible modes:
+  // - Terminate: The TLS session between the downstream client
+  //   and the Gateway is terminated at the Gateway.
+  // - Passthrough: The TLS session NOT terminated by the Gateway. This
```

Comment:
The godocs should states that all other TLSConfig fields are ignored when mode = Passthrough